dependencies.yaml

```
@@ -88,6 +88,11 @@ files:
       key: test
     includes:
       - test_python
+  test_java:
+    output: none
+    includes:
+      - cuda
+      - test_java
```

I'm guessing you'll need CMake and C++ compilers? You'll also need `cuda_version` to constrain which nvcc and nvcomp versions you get.
```suggestion
    includes:
      - build-universal
      - build-cpp
      - cuda_version
      - cuda
      - test_java
```